scripts/shellcheck.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -e
+
+docker run -v "$(pwd)":/wrk -w /wrk koalaman/shellcheck scripts/*.sh
```

Comment:
We should also include `go generate pkg/types/main.go` which generates `pkg/types/create*_string.go`. Note that it requires `go get golang.org/x/tools/cmd/stringer` before-hand.

Reply:
Awesome! Done.